jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -19,7 +19,7 @@ Use parallelism and test splitting to:
 ## Introduction
 {: #introduction }
 
-Pipelines are often configured so that each time code is committed a set of tests are run. The more tests your project has, the longer it will take for them to complete using a single compute resource. To reduce this time, you can split your tests and run them across multiple, parallel-running execution environments. Test splitting is a great way to speed up the testing portion of your CI/CD pipeline.
+Pipelines are often configured so that each time code is committed, a set of tests are run. The more tests your project has, the longer it will take for them to complete using a single compute resource. To reduce this time, you can split your tests and run them across multiple, parallel-running execution environments. Test splitting is a great way to speed up the testing portion of your CI/CD pipeline.
```

Comment:
```suggestion
Pipelines are often configured so that a set of tests are run each time code is committed. The more tests your project has, the longer it will take for them to complete using a single compute resource. To reduce this time, you can split your tests and run them across multiple, parallel-running execution environments. Test splitting is a great way to speed up the testing portion of your CI/CD pipeline.
```

Reply:
@AfiMaameDufie thanks so much for pointing out this sentence has some issues. I think this might be clearer?

Reply:
Yep, it's way clearer now. Thank you :)